src/front/app/containers/SampleListUsersContainer.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file Sample list users Container.
+ * Shows how to implement react with list users query.
+ */
+import { compose, graphql } from 'react-apollo';
+import SampleListUsers from '../components/users/SampleListUsers';
+import { Users } from '../queries/users';
+
+const withListUsers = graphql(Users, {
+  // Use options to pass variables.
+  // It takes `ownProps` as params.
+  options: () => ({
+    variables: {
+      limit: 25,
+    },
+  }),
```

Comment:
this `option` variable is stated to take `ownProps` as a param, but at least I don't see it. In order to make this easier to understand, it'd be good if you either remove the comment or add the props inside.

Reply:
If I do, it'll blow up :) You can only pass variables that are accepted by the query